### DISCUSSION TITLE
VCTK dataset / model licence question

### DISCUSSION THREAD

**Discussion by jeetuv** [coqui-ai/TTS#2477] [General Q&A]
As far as I understand, the VCTK dataset  (https://datashare.ed.ac.uk/handle/10283/3443) is released under Creative Commons License: Attribution 4.0 International license. 

However, as per the ._models.json_ file, some models under the VCTK category, like `vctk -> vits` have Apache 2.0 license.

1. My first question is - can models created from a CC-BY license, be released under the more liberal license? Do dataset licenses apply only to derived dataset and not to models inferred from the dataset?

2. Second question is - do voiceovers generated from a model, fall under the same license? Or is the model license applicable only to derived models?  (For example - can one use `vtck -> fast_pitch`, which has "CC BY-NC-ND 4.0" license, for generating audio tracks, which are used for commercial purpose without attribution? Can voice tracks generated from vctk->vits be used commercially without attribution?).

**Comment by kdavis-coqui** [comment]
As we can't offer legal advice, I'd suggest consulting with a legal professional for all such licensing questions.
